src/components/Radio/orderRadio.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import { Radio, Space } from "antd";
+
+const OrderRadio = ({value, setValue}) => {
+    const onChangeRadio = e => {
+        setValue(e.target.value);
+    }
+
+    return(
+        <Radio.Group
+            onChange={onChangeRadio}
+            value={value}
+            style={{
+                fontSize: '30px',
+            }}
+        >
+            <Space direction="vertical">
+                <Radio 
```

Comment:
We can remove the ones that are no supported in the server-side, we only support `asc` and `desc`